src/tests/ov_helpers/ov_models/include/ov_models/utils/ov_helpers.hpp

```diff
@@ -80,39 +80,18 @@ using ov::test::utils::LogicalTypes;
 using ov::test::utils::SqueezeOpType;
 using ov::test::utils::MinMaxOpType;
 
-enum QuantizationGranularity {
```

Comment:
Don't remove this implementation because it's used in separate repository and causes build error

Reply:
@vurusovs @praasz Had created another PR to fix the error, need VPUX Plugin guys help review & approve.
https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/7282